src/TikaOnDotNet.Tests/text_extraction.cs

```diff
@@ -201,30 +206,52 @@ public void should_extract_msg()
         }
 
         [Test]
-        public void should_ocr_with_tesseract()
+        public void should_ocr_tif_with_tesseract()
         {
-            const string tesseractPath = @"c:\Program Files (x86)\Tesseract-OCR";
-
-            if (!File.Exists(Path.Combine(tesseractPath, "tesseract.exe")))
-                Assert.Ignore("Tesseract not found in the path '" + tesseractPath + "'");
+            if (!File.Exists(Path.Combine(TesseractPath, "tesseract.exe")))

```

Comment:
What do you think about updating the build automation so that Tesseract is present? Otherwise these tests will always be ignored by CI. Paket could pull it down for us if there is no Nuget with the `tesseract.exe`. Check out  `package.dependencies` to see how it is done for IKVM.